README.md

```diff
@@ -1895,7 +1895,7 @@ no parameters.
   ```
 
 * <a name="camelcase-classes"></a>
-  Use `CamelCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
+  Use `PascalCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
```

Comment:
I'm aware of the term `PascalCase`, but I'm pretty sure most people are not - therefore the decision to use `CamelCase` (although it's actually `camelCase`).